build-logic/project-plugins/src/main/kotlin/com.hedera.hashgraph.jpms-modules.gradle.kts

```diff
@@ -145,8 +144,13 @@ extraJavaModuleInfo {
     module("io.grpc:grpc-services", "grpc.services")
     module("io.grpc:grpc-protobuf", "grpc.protobuf")
     module("io.grpc:grpc-protobuf-lite", "grpc.protobuf.lite")
-    module("javax.annotation:javax.annotation-api", "java.annotation")
     module("com.github.spotbugs:spotbugs-annotations", "com.github.spotbugs.annotations")
+    module("com.google.code.findbugs:jsr305", "java.annotation") {
+        exportAllPackages()
+        mergeJar("javax.annotation:javax.annotation-api")
```

Comment:
The changes here are necessary because the strict evaluation of annotation processors revealed some missing "compile time only" classes that annotation processors see.